commons/pom.xml

```diff
@@ -17,6 +17,7 @@
       <dependency>
          <groupId>com.github.ben-manes.caffeine</groupId>
          <artifactId>caffeine</artifactId>
+         <optional>true</optional>
```

Comment:
In other words: it would be great if infinispan-commons would avoid having this dependency, then we'd have reassurances from the compiler.